pkg/executor/ddl.go

```diff
@@ -229,12 +210,6 @@ func (e *DDLExec) Next(ctx context.Context, _ *chunk.Chunk) (err error) {
 		err = e.executeAlterResourceGroup(x)
 	}
 	if err != nil {
-		// If the owner return ErrTableNotExists error when running this DDL, it may be caused by schema changed,
```

Comment:
seems the old code converts ErrTableNotExists to "inforschema is changed" and this PR wants to skip converting for ErrTableExists. Can we add a `!infoschema.ErrTableExists.Equal(err)` to `!isDDLJobInQueue` case?

(I mean 
```go
if (isDDLJobInQueue && infoschema.ErrTableNotExists.Equal(err)) || (!isDDLJobInQueue && !infoschema.ErrTableExists.Equal(err))
```
)